docs/source/index.rst

```diff
@@ -118,9 +118,13 @@ Autodetection heuristics
   * functions that have an attribute ``__returns_contextmanager__``
     with a truthy value.
 
-* ``:async-with:`` is autodetected for functions that have an
-  attribute ``__returns_acontextmanager__`` (note the ``a``) with a
-  truthy value.
+* ``:async-with:`` is autodetected for:
+
+  * functions that have an attribute ``__returns_acontextmanager__`` 
+    (note the ``a``) with a truthy value.
+  
+  * functions decorated with `contextlib.asynccontextmanager
```

Comment:
Let's make the order of these bullets consistent with the synchronous context manager case above.